### PR TITLE
Update app_to_web.md

### PR DESCRIPTION
**Pull request by johnivdel**
Remove deprecated intent based explainer in favor of new approach

### CODE REVIEW

**Comment by johnivdel** [comment]
@linnan-github please review

File: app_to_web.md [hunk 1]
```diff
@@ -14,214 +14,69 @@ Please file [GitHub issues](https://github.com/WICG/conversion-measurement-api/i
...
 
-Any browser on Android could implement this API. This proposal was inspired by Safari's [App-to-web support in Private Click Measurement](https://webkit.org/blog/11529/introducing-private-click-measurement-pcm/#:~:text=App-to-Web).
-
+This proposal was inspired by Safari's [App-to-web support in Private Click Measurement](https://webkit.org/blog/11529/introducing-private-click-measurement-pcm/#:~:text=App-to-Web) and the [Android Privacy Sandbox](https://developer.android.com/design-for-safety/privacy-sandbox/attribution) proposal.
 
+The proposal here takes advantage of OS-level support for attribution. In particular, it gives the developer an option to allow events on the mobile web to be joinable with events in [Android’s Privacy Sandbox](https://developer.android.com/design-for-safety/privacy-sandbox/attribution), although support for other platforms could also be implemented.
 ## Goals
```
Comment (linnan-github) [inline app_to_web.md:20]:
```

```

Comment (linnan-github) [inline app_to_web.md:20]:
```suggestion

## Goals
```

File: app_to_web.md [hunk 2]
```diff
@@ -14,214 +14,69 @@ Please file [GitHub issues](https://github.com/WICG/conversion-measurement-api/i
...
-*   Support measurement of ads that show up within apps that later convert in the browser
-*   Support measurement of ads within embedded webview that later convert in the browser
+*   Support measurement of ads that show up within apps that later convert on the web
+*   Support measurement of ads that show up within the browser that later convert on the web or in an app / app install
 *   Support both clicks and views
-
-
+*   Proposal is generic enough to be implemented by any browser
+*   While focusing initially on Android support, the proposal should be generic enough to be supported on other platforms
 ## API changes
```
Comment (linnan-github) [inline app_to_web.md:27]:
```suggestion

## API changes
```

File: app_to_web.md [hunk 3]
```diff
@@ -14,214 +14,69 @@ Please file [GitHub issues](https://github.com/WICG/conversion-measurement-api/i
...
+See Android's [Attribution reporting: cross app and web measurement proposal](https://developer.android.com/design-for-safety/privacy-sandbox/attribution-app-to-web) for one example of as OS API that a browser can integrate with to do cross app and web measurement.
```
Comment (linnan-github) [inline app_to_web.md:28]: "as OS API" or "an OS API"?

File: app_to_web.md [hunk 4]
```diff
@@ -14,214 +14,69 @@ Please file [GitHub issues](https://github.com/WICG/conversion-measurement-api/i
...
-<!-- Click-through attributions: -->
-<a 
-  href="https://advertiser.example/buy-shoes"
-  attributionsourceeventid="123456"
-  attributionreportto="https://reporter.example"
-  attributiondestination="https://advertiser.example"
-  attributionexpiry=604800000>
-  Click me!
-</a>
+Attribution-Reporting-Support: os=?1 ; web=?1
```
Comment (linnan-github) [inline app_to_web.md:32]:
```suggestion
Attribution-Reporting-Support: os=?1; web=?1
```

File: app_to_web.md [hunk 5]
```diff
@@ -14,214 +14,69 @@ Please file [GitHub issues](https://github.com/WICG/conversion-measurement-api/i
...
-try {
-  client.insert(providerUri, attributionValues);
-} catch (Exception e) {
-  // Always catch exceptions thrown by ContentProviders so an error in the browser cannot crash your app.
-}
-
-// The client should be closed when no longer needed (eg. your app is no longer foreground).
-client.close();
+// Registers a source against a native OS attribution API
+Attribution-Reporting-Register-OS-Source: “https://adtech.example/register-android-source?...” ; os-destination=<os destination> ; web-destination=<web destination>
```
Comment (linnan-github) [inline app_to_web.md:37]:
```suggestion
Attribution-Reporting-Register-OS-Source: “https://adtech.example/register-android-source?...”; os-destination=<os destination>; web-destination=<web destination>

```

File: app_to_web.md [hunk 6]
```diff
@@ -14,214 +14,69 @@ Please file [GitHub issues](https://github.com/WICG/conversion-measurement-api/i
...
+This proposal explicitly links data from the web with data from apps. It enables apps to learn coarse user behavior patterns in the browser in the same way that the existing Attribution Reporting API allows websites to learn coarse user behavior patterns in the browser. In other words, we can safely think of an app as a particular "kind of website", and we can share data to apps that we’d be comfortable sharing to a (cross-site) website.
 
+However, there are some “privacy budget” / rate-limit consequences to expanding support to the OS. In particular, if the OS maintains a separate budget than the browser, an adversary could in theory consume both the browser-local budget and the OS system budget, leaking more than if only the browser-local budget were used. Mitigations for this issue include stricter budgets overall, or being more restrictive about entities that can register both OS-level events and browser-local events.
 
+Additionally, this proposal sends user information from the browser to the operating system. The OS will be responsible for integrating with browser user controls such as those for clearing browsing history.
+### New fingerprint vector
```
Comment (linnan-github) [inline app_to_web.md:64]:
```suggestion

### New fingerprint vector
```

File: app_to_web.md [hunk 7]
```diff
@@ -14,214 +14,69 @@ Please file [GitHub issues](https://github.com/WICG/conversion-measurement-api/i
...
+The new request header we would add to the API, `Attribution-Reporting-OS-Support: ?1` adds an additional fingerprinting vector letting an adversary know if the underlying platform supports attribution reporting.
+### User Control
```
Comment (linnan-github) [inline app_to_web.md:66]:
```suggestion

### User Control
```

File: app_to_web.md [hunk 8]
```diff
@@ -14,214 +14,69 @@ Please file [GitHub issues](https://github.com/WICG/conversion-measurement-api/i
...
+Attribution between two websites within the browser doesn't require specific OS support. However, attribution of web to app or app to web will require (1) that a user's device supports both a valid version of the browser and a valid version of the operating system and (2) that the user has not disabled the Attribution Reporting API for either the browser or the platform. If the user has opted out of the API in either the browser or at the OS level, app to web or web to app attribution will not be enabled.
 ## Security considerations
```
Comment (linnan-github) [inline app_to_web.md:68]:
```suggestion

## Security considerations
```

**Review by linnan-github — APPROVED** [review]
LGTM % nits

**Pull request merged**